### ISSUE TITLE
💅 (lint/correctness/useJsxKeyInIterable) False Positive Error

### ISSUE THREAD

**Issue by ImBIOS** [biomejs/biome#2518]
### Environment information

```bash
CLI:
  Version:                      1.6.4
  Color support:                true

Platform:
  CPU Architecture:             x86_64
  OS:                           linux

Environment:
  BIOME_LOG_DIR:                unset
  NO_COLOR:                     unset
  TERM:                         "xterm-256color"
  JS_RUNTIME_VERSION:           "v18.20.2"
  JS_RUNTIME_NAME:              "node"
  NODE_PACKAGE_MANAGER:         "bun/1.1.4"

Biome Configuration:
  Status:                       Loaded successfully
  Formatter disabled:           false
  Linter disabled:              false
  Organize imports disabled:    true
  VCS disabled:                 false

Linter:
  Recommended:                  true
  All:                          false
  Rules:
Workspace:
  Open Documents:               0
```

### Rule name

lint/correctness/useJsxKeyInIterable

### Code

```side-nav.tsx
"use client";

import { ChevronDownIcon } from "@radix-ui/react-icons";
import Link from "next/link";
import { usePathname } from "next/navigation";
import { useEffect, useState } from "react";

import { buttonVariants } from "~/components/ui/button";
import { useSidebar } from "~/lib/hooks/use-sidebar";
import type { NavItem } from "~/lib/types";
import { cn } from "~/lib/utils";

import {
	Accordion,
	AccordionContent,
	AccordionItem,
	AccordionTrigger,
} from "./subnav-accordion";

interface SideNavProps {
	items: NavItem[];
	setOpen?: (open: boolean) => void;
	className?: string;
}

function SideNav({ items, setOpen, className }: SideNavProps) {
	const path = usePathname();
	const { isOpen } = useSidebar();
	const [openItem, setOpenItem] = useState("");
	const [lastOpenItem, setLastOpenItem] = useState("");

	useEffect(() => {
		if (isOpen) {
			setOpenItem(lastOpenItem);
		} else {
			setLastOpenItem(openItem);
			setOpenItem("");
		}
	}, [isOpen, lastOpenItem, openItem]);

	return (
		<nav className="space-y-2">
			{items.map((item) =>
				item.isChidren ? (
					<Accordion
						type="single"
						collapsible
						className="space-y-2"
						key={item.title}
						value={openItem}
						onValueChange={setOpenItem}
					>
						<AccordionItem value={item.title} className="border-none ">
							<AccordionTrigger
								className={cn(
									buttonVariants({ variant: "ghost" }),
									"group relative flex h-12 justify-between px-4 py-2 text-base duration-200 hover:bg-muted hover:no-underline",
								)}
							>
								<div>
									<item.icon className={cn("size-5", item.color)} />
								</div>
								<div
									className={cn(
										"absolute left-12 text-base duration-200 ",
										!isOpen && className,
									)}
								>
									{item.title}
								</div>

								{isOpen && (
									<ChevronDownIcon className="size-4 shrink-0 text-muted-foreground transition-transform duration-200" />
								)}
							</AccordionTrigger>
							<AccordionContent className="mt-2 space-y-4 pb-1">
								{item.children?.map((child) => (
									<Link
										key={child.title}
										href={child.href}
										onClick={() => {
											if (setOpen) setOpen(false);
										}}
										className={cn(
											buttonVariants({ variant: "ghost" }),
											"group relative flex h-12 justify-start gap-x-3",
											path === child.href &&
												"bg-muted font-bold hover:bg-muted",
										)}
									>
										<child.icon className={cn("size-5", child.color)} />
										<div
											className={cn(
												"absolute left-12 text-base duration-200",
												!isOpen && className,
											)}
										>
											{child.title}
										</div>
									</Link>
								))}
							</AccordionContent>
						</AccordionItem>
					</Accordion>
				) : (
					<Link
						key={item.title}
						href={item.href}
						onClick={() => {
							if (setOpen) setOpen(false);
						}}
						className={cn(
							buttonVariants({ variant: "ghost" }),
							"group relative flex h-12 justify-start",
							path === item.href && "bg-muted font-bold hover:bg-muted",
						)}
					>
						<item.icon className={cn("size-5", item.color)} />
						<span
							className={cn(
								"absolute left-12 text-base duration-200",
								!isOpen && className,
							)}
						>
							{item.title}
						</span>
					</Link>
				),
			)}
		</nav>
	);
}

export default SideNav;
```

### Playground link

[click here](https://biomejs.dev/playground/?code=IgB1AHMAZQAgAGMAbABpAGUAbgB0ACIAOwAKAAoAaQBtAHAAbwByAHQAIAB7ACAAQwBoAGUAdgByAG8AbgBEAG8AdwBuAEkAYwBvAG4AIAB9ACAAZgByAG8AbQAgACIAQAByAGEAZABpAHgALQB1AGkALwByAGUAYQBjAHQALQBpAGMAbwBuAHMAIgA7AAoAaQBtAHAAbwByAHQAIABMAGkAbgBrACAAZgByAG8AbQAgACIAbgBlAHgAdAAvAGwAaQBuAGsAIgA7AAoAaQBtAHAAbwByAHQAIAB7ACAAdQBzAGUAUABhAHQAaABuAGEAbQBlACAAfQAgAGYAcgBvAG0AIAAiAG4AZQB4AHQALwBuAGEAdgBpAGcAYQB0AGkAbwBuACIAOwAKAGkAbQBwAG8AcgB0ACAAewAgAHUAcwBlAEUAZgBmAGUAYwB0ACwAIAB1AHMAZQBTAHQAYQB0AGUAIAB9ACAAZgByAG8AbQAgACIAcgBlAGEAYwB0ACIAOwAKAAoAaQBtAHAAbwByAHQAIAB7ACAAYgB1AHQAdABvAG4AVgBhAHIAaQBhAG4AdABzACAAfQAgAGYAcgBvAG0AIAAiAH4ALwBjAG8AbQBwAG8AbgBlAG4AdABzAC8AdQBpAC8AYgB1AHQAdABvAG4AIgA7AAoAaQBtAHAAbwByAHQAIAB7ACAAdQBzAGUAUwBpAGQAZQBiAGEAcgAgAH0AIABmAHIAbwBtACAAIgB+AC8AbABpAGIALwBoAG8AbwBrAHMALwB1AHMAZQAtAHMAaQBkAGUAYgBhAHIAIgA7AAoAaQBtAHAAbwByAHQAIAB0AHkAcABlACAAewAgAE4AYQB2AEkAdABlAG0AIAB9ACAAZgByAG8AbQAgACIAfgAvAGwAaQBiAC8AdAB5AHAAZQBzACIAOwAKAGkAbQBwAG8AcgB0ACAAewAgAGMAbgAgAH0AIABmAHIAbwBtACAAIgB+AC8AbABpAGIALwB1AHQAaQBsAHMAIgA7AAoACgBpAG0AcABvAHIAdAAgAHsACgAJAEEAYwBjAG8AcgBkAGkAbwBuACwACgAJAEEAYwBjAG8AcgBkAGkAbwBuAEMAbwBuAHQAZQBuAHQALAAKAAkAQQBjAGMAbwByAGQAaQBvAG4ASQB0AGUAbQAsAAoACQBBAGMAYwBvAHIAZABpAG8AbgBUAHIAaQBnAGcAZQByACwACgB9ACAAZgByAG8AbQAgACIALgAvAHMAdQBiAG4AYQB2AC0AYQBjAGMAbwByAGQAaQBvAG4AIgA7AAoACgBpAG4AdABlAHIAZgBhAGMAZQAgAFMAaQBkAGUATgBhAHYAUAByAG8AcABzACAAewAKAAkAaQB0AGUAbQBzADoAIABOAGEAdgBJAHQAZQBtAFsAXQA7AAoACQBzAGUAdABPAHAAZQBuAD8AOgAgACgAbwBwAGUAbgA6ACAAYgBvAG8AbABlAGEAbgApACAAPQA+ACAAdgBvAGkAZAA7AAoACQBjAGwAYQBzAHMATgBhAG0AZQA/ADoAIABzAHQAcgBpAG4AZwA7AAoAfQAKAAoAZgB1AG4AYwB0AGkAbwBuACAAUwBpAGQAZQBOAGEAdgAoAHsAIABpAHQAZQBtAHMALAAgAHMAZQB0AE8AcABlAG4ALAAgAGMAbABhAHMAcwBOAGEAbQBlACAAfQA6ACAAUwBpAGQAZQBOAGEAdgBQAHIAbwBwAHMAKQAgAHsACgAJAGMAbwBuAHMAdAAgAHAAYQB0AGgAIAA9ACAAdQBzAGUAUABhAHQAaABuAGEAbQBlACgAKQA7AAoACQBjAG8AbgBzAHQAIAB7ACAAaQBzAE8AcABlAG4AIAB9ACAAPQAgAHUAcwBlAFMAaQBkAGUAYgBhAHIAKAApADsACgAJAGMAbwBuAHMAdAAgAFsAbwBwAGUAbgBJAHQAZQBtACwAIABzAGUAdABPAHAAZQBuAEkAdABlAG0AXQAgAD0AIAB1AHMAZQBTAHQAYQB0AGUAKAAiACIAKQA7AAoACQBjAG8AbgBzAHQAIABbAGwAYQBzAHQATwBwAGUAbgBJAHQAZQBtACwAIABzAGUAdABMAGEAcwB0AE8AcABlAG4ASQB0AGUAbQBdACAAPQAgAHUAcwBlAFMAdABhAHQAZQAoACIAIgApADsACgAKAAkAdQBzAGUARQBmAGYAZQBjAHQAKAAoACkAIAA9AD4AIAB7AAoACQAJAGkAZgAgACgAaQBzAE8AcABlAG4AKQAgAHsACgAJAAkACQBzAGUAdABPAHAAZQBuAEkAdABlAG0AKABsAGEAcwB0AE8AcABlAG4ASQB0AGUAbQApADsACgAJAAkAfQAgAGUAbABzAGUAIAB7AAoACQAJAAkAcwBlAHQATABhAHMAdABPAHAAZQBuAEkAdABlAG0AKABvAHAAZQBuAEkAdABlAG0AKQA7AAoACQAJAAkAcwBlAHQATwBwAGUAbgBJAHQAZQBtACgAIgAiACkAOwAKAAkACQB9AAoACQB9ACwAIABbAGkAcwBPAHAAZQBuACwAIABsAGEAcwB0AE8AcABlAG4ASQB0AGUAbQAsACAAbwBwAGUAbgBJAHQAZQBtAF0AKQA7AAoACgAJAHIAZQB0AHUAcgBuACAAKAAKAAkACQA8AG4AYQB2ACAAYwBsAGEAcwBzAE4AYQBtAGUAPQAiAHMAcABhAGMAZQAtAHkALQAyACIAPgAKAAkACQAJAHsAaQB0AGUAbQBzAC4AbQBhAHAAKAAoAGkAdABlAG0AKQAgAD0APgAKAAkACQAJAAkAaQB0AGUAbQAuAGkAcwBDAGgAaQBkAHIAZQBuACAAPwAgACgACgAJAAkACQAJAAkAPABBAGMAYwBvAHIAZABpAG8AbgAKAAkACQAJAAkACQAJAHQAeQBwAGUAPQAiAHMAaQBuAGcAbABlACIACgAJAAkACQAJAAkACQBjAG8AbABsAGEAcABzAGkAYgBsAGUACgAJAAkACQAJAAkACQBjAGwAYQBzAHMATgBhAG0AZQA9ACIAcwBwAGEAYwBlAC0AeQAtADIAIgAKAAkACQAJAAkACQAJAGsAZQB5AD0AewBpAHQAZQBtAC4AdABpAHQAbABlAH0ACgAJAAkACQAJAAkACQB2AGEAbAB1AGUAPQB7AG8AcABlAG4ASQB0AGUAbQB9AAoACQAJAAkACQAJAAkAbwBuAFYAYQBsAHUAZQBDAGgAYQBuAGcAZQA9AHsAcwBlAHQATwBwAGUAbgBJAHQAZQBtAH0ACgAJAAkACQAJAAkAPgAKAAkACQAJAAkACQAJADwAQQBjAGMAbwByAGQAaQBvAG4ASQB0AGUAbQAgAHYAYQBsAHUAZQA9AHsAaQB0AGUAbQAuAHQAaQB0AGwAZQB9ACAAYwBsAGEAcwBzAE4AYQBtAGUAPQAiAGIAbwByAGQAZQByAC0AbgBvAG4AZQAgACIAPgAKAAkACQAJAAkACQAJAAkAPABBAGMAYwBvAHIAZABpAG8AbgBUAHIAaQBnAGcAZQByAAoACQAJAAkACQAJAAkACQAJAGMAbABhAHMAcwBOAGEAbQBlAD0AewBjAG4AKAAKAAkACQAJAAkACQAJAAkACQAJAGIAdQB0AHQAbwBuAFYAYQByAGkAYQBuAHQAcwAoAHsAIAB2AGEAcgBpAGEAbgB0ADoAIAAiAGcAaABvAHMAdAAiACAAfQApACwACgAJAAkACQAJAAkACQAJAAkACQAiAGcAcgBvAHUAcAAgAHIAZQBsAGEAdABpAHYAZQAgAGYAbABlAHgAIABoAC0AMQAyACAAagB1AHMAdABpAGYAeQAtAGIAZQB0AHcAZQBlAG4AIABwAHgALQA0ACAAcAB5AC0AMgAgAHQAZQB4AHQALQBiAGEAcwBlACAAZAB1AHIAYQB0AGkAbwBuAC0AMgAwADAAIABoAG8AdgBlAHIAOgBiAGcALQBtAHUAdABlAGQAIABoAG8AdgBlAHIAOgBuAG8ALQB1AG4AZABlAHIAbABpAG4AZQAiACwACgAJAAkACQAJAAkACQAJAAkAKQB9AAoACQAJAAkACQAJAAkACQA+AAoACQAJAAkACQAJAAkACQAJADwAZABpAHYAPgAKAAkACQAJAAkACQAJAAkACQAJADwAaQB0AGUAbQAuAGkAYwBvAG4AIABjAGwAYQBzAHMATgBhAG0AZQA9AHsAYwBuACgAIgBzAGkAegBlAC0ANQAiACwAIABpAHQAZQBtAC4AYwBvAGwAbwByACkAfQAgAC8APgAKAAkACQAJAAkACQAJAAkACQA8AC8AZABpAHYAPgAKAAkACQAJAAkACQAJAAkACQA8AGQAaQB2AAoACQAJAAkACQAJAAkACQAJAAkAYwBsAGEAcwBzAE4AYQBtAGUAPQB7AGMAbgAoAAoACQAJAAkACQAJAAkACQAJAAkACQAiAGEAYgBzAG8AbAB1AHQAZQAgAGwAZQBmAHQALQAxADIAIAB0AGUAeAB0AC0AYgBhAHMAZQAgAGQAdQByAGEAdABpAG8AbgAtADIAMAAwACAAIgAsAAoACQAJAAkACQAJAAkACQAJAAkACQAhAGkAcwBPAHAAZQBuACAAJgAmACAAYwBsAGEAcwBzAE4AYQBtAGUALAAKAAkACQAJAAkACQAJAAkACQAJACkAfQAKAAkACQAJAAkACQAJAAkACQA+AAoACQAJAAkACQAJAAkACQAJAAkAewBpAHQAZQBtAC4AdABpAHQAbABlAH0ACgAJAAkACQAJAAkACQAJAAkAPAAvAGQAaQB2AD4ACgAKAAkACQAJAAkACQAJAAkACQB7AGkAcwBPAHAAZQBuACAAJgAmACAAKAAKAAkACQAJAAkACQAJAAkACQAJADwAQwBoAGUAdgByAG8AbgBEAG8AdwBuAEkAYwBvAG4AIABjAGwAYQBzAHMATgBhAG0AZQA9ACIAcwBpAHoAZQAtADQAIABzAGgAcgBpAG4AawAtADAAIAB0AGUAeAB0AC0AbQB1AHQAZQBkAC0AZgBvAHIAZQBnAHIAbwB1AG4AZAAgAHQAcgBhAG4AcwBpAHQAaQBvAG4ALQB0AHIAYQBuAHMAZgBvAHIAbQAgAGQAdQByAGEAdABpAG8AbgAtADIAMAAwACIAIAAvAD4ACgAJAAkACQAJAAkACQAJAAkAKQB9AAoACQAJAAkACQAJAAkACQA8AC8AQQBjAGMAbwByAGQAaQBvAG4AVAByAGkAZwBnAGUAcgA+AAoACQAJAAkACQAJAAkACQA8AEEAYwBjAG8AcgBkAGkAbwBuAEMAbwBuAHQAZQBuAHQAIABjAGwAYQBzAHMATgBhAG0AZQA9ACIAbQB0AC0AMgAgAHMAcABhAGMAZQAtAHkALQA0ACAAcABiAC0AMQAiAD4ACgAJAAkACQAJAAkACQAJAAkAewBpAHQAZQBtAC4AYwBoAGkAbABkAHIAZQBuAD8ALgBtAGEAcAAoACgAYwBoAGkAbABkACkAIAA9AD4AIAAoAAoACQAJAAkACQAJAAkACQAJAAkAPABMAGkAbgBrAAoACQAJAAkACQAJAAkACQAJAAkACQBrAGUAeQA9AHsAYwBoAGkAbABkAC4AdABpAHQAbABlAH0ACgAJAAkACQAJAAkACQAJAAkACQAJAGgAcgBlAGYAPQB7AGMAaABpAGwAZAAuAGgAcgBlAGYAfQAKAAkACQAJAAkACQAJAAkACQAJAAkAbwBuAEMAbABpAGMAawA9AHsAKAApACAAPQA+ACAAewAKAAkACQAJAAkACQAJAAkACQAJAAkACQBpAGYAIAAoAHMAZQB0AE8AcABlAG4AKQAgAHMAZQB0AE8AcABlAG4AKABmAGEAbABzAGUAKQA7AAoACQAJAAkACQAJAAkACQAJAAkACQB9AH0ACgAJAAkACQAJAAkACQAJAAkACQAJAGMAbABhAHMAcwBOAGEAbQBlAD0AewBjAG4AKAAKAAkACQAJAAkACQAJAAkACQAJAAkACQBiAHUAdAB0AG8AbgBWAGEAcgBpAGEAbgB0AHMAKAB7ACAAdgBhAHIAaQBhAG4AdAA6ACAAIgBnAGgAbwBzAHQAIgAgAH0AKQAsAAoACQAJAAkACQAJAAkACQAJAAkACQAJACIAZwByAG8AdQBwACAAcgBlAGwAYQB0AGkAdgBlACAAZgBsAGUAeAAgAGgALQAxADIAIABqAHUAcwB0AGkAZgB5AC0AcwB0AGEAcgB0ACAAZwBhAHAALQB4AC0AMwAiACwACgAJAAkACQAJAAkACQAJAAkACQAJAAkAcABhAHQAaAAgAD0APQA9ACAAYwBoAGkAbABkAC4AaAByAGUAZgAgACYAJgAKAAkACQAJAAkACQAJAAkACQAJAAkACQAJACIAYgBnAC0AbQB1AHQAZQBkACAAZgBvAG4AdAAtAGIAbwBsAGQAIABoAG8AdgBlAHIAOgBiAGcALQBtAHUAdABlAGQAIgAsAAoACQAJAAkACQAJAAkACQAJAAkACQApAH0ACgAJAAkACQAJAAkACQAJAAkACQA+AAoACQAJAAkACQAJAAkACQAJAAkACQA8AGMAaABpAGwAZAAuAGkAYwBvAG4AIABjAGwAYQBzAHMATgBhAG0AZQA9AHsAYwBuACgAIgBzAGkAegBlAC0ANQAiACwAIABjAGgAaQBsAGQALgBjAG8AbABvAHIAKQB9ACAALwA+AAoACQAJAAkACQAJAAkACQAJAAkACQA8AGQAaQB2AAoACQAJAAkACQAJAAkACQAJAAkACQAJAGMAbABhAHMAcwBOAGEAbQBlAD0AewBjAG4AKAAKAAkACQAJAAkACQAJAAkACQAJAAkACQAJACIAYQBiAHMAbwBsAHUAdABlACAAbABlAGYAdAAtADEAMgAgAHQAZQB4AHQALQBiAGEAcwBlACAAZAB1AHIAYQB0AGkAbwBuAC0AMgAwADAAIgAsAAoACQAJAAkACQAJAAkACQAJAAkACQAJAAkAIQBpAHMATwBwAGUAbgAgACYAJgAgAGMAbABhAHMAcwBOAGEAbQBlACwACgAJAAkACQAJAAkACQAJAAkACQAJAAkAKQB9AAoACQAJAAkACQAJAAkACQAJAAkACQA+AAoACQAJAAkACQAJAAkACQAJAAkACQAJAHsAYwBoAGkAbABkAC4AdABpAHQAbABlAH0ACgAJAAkACQAJAAkACQAJAAkACQAJADwALwBkAGkAdgA+AAoACQAJAAkACQAJAAkACQAJAAkAPAAvAEwAaQBuAGsAPgAKAAkACQAJAAkACQAJAAkACQApACkAfQAKAAkACQAJAAkACQAJAAkAPAAvAEEAYwBjAG8AcgBkAGkAbwBuAEMAbwBuAHQAZQBuAHQAPgAKAAkACQAJAAkACQAJADwALwBBAGMAYwBvAHIAZABpAG8AbgBJAHQAZQBtAD4ACgAJAAkACQAJAAkAPAAvAEEAYwBjAG8AcgBkAGkAbwBuAD4ACgAJAAkACQAJACkAIAA6ACAAKAAKAAkACQAJAAkACQA8AEwAaQBuAGsACgAJAAkACQAJAAkACQBrAGUAeQA9AHsAaQB0AGUAbQAuAHQAaQB0AGwAZQB9AAoACQAJAAkACQAJAAkAaAByAGUAZgA9AHsAaQB0AGUAbQAuAGgAcgBlAGYAfQAKAAkACQAJAAkACQAJAG8AbgBDAGwAaQBjAGsAPQB7ACgAKQAgAD0APgAgAHsACgAJAAkACQAJAAkACQAJAGkAZgAgACgAcwBlAHQATwBwAGUAbgApACAAcwBlAHQATwBwAGUAbgAoAGYAYQBsAHMAZQApADsACgAJAAkACQAJAAkACQB9AH0ACgAJAAkACQAJAAkACQBjAGwAYQBzAHMATgBhAG0AZQA9AHsAYwBuACgACgAJAAkACQAJAAkACQAJAGIAdQB0AHQAbwBuAFYAYQByAGkAYQBuAHQAcwAoAHsAIAB2AGEAcgBpAGEAbgB0ADoAIAAiAGcAaABvAHMAdAAiACAAfQApACwACgAJAAkACQAJAAkACQAJACIAZwByAG8AdQBwACAAcgBlAGwAYQB0AGkAdgBlACAAZgBsAGUAeAAgAGgALQAxADIAIABqAHUAcwB0AGkAZgB5AC0AcwB0AGEAcgB0ACIALAAKAAkACQAJAAkACQAJAAkAcABhAHQAaAAgAD0APQA9ACAAaQB0AGUAbQAuAGgAcgBlAGYAIAAmACYAIAAiAGIAZwAtAG0AdQB0AGUAZAAgAGYAbwBuAHQALQBiAG8AbABkACAAaABvAHYAZQByADoAYgBnAC0AbQB1AHQAZQBkACIALAAKAAkACQAJAAkACQAJACkAfQAKAAkACQAJAAkACQA+AAoACQAJAAkACQAJAAkAPABpAHQAZQBtAC4AaQBjAG8AbgAgAGMAbABhAHMAcwBOAGEAbQBlAD0AewBjAG4AKAAiAHMAaQB6AGUALQA1ACIALAAgAGkAdABlAG0ALgBjAG8AbABvAHIAKQB9ACAALwA+AAoACQAJAAkACQAJAAkAPABzAHAAYQBuAAoACQAJAAkACQAJAAkACQBjAGwAYQBzAHMATgBhAG0AZQA9AHsAYwBuACgACgAJAAkACQAJAAkACQAJAAkAIgBhAGIAcwBvAGwAdQB0AGUAIABsAGUAZgB0AC0AMQAyACAAdABlAHgAdAAtAGIAYQBzAGUAIABkAHUAcgBhAHQAaQBvAG4ALQAyADAAMAAiACwACgAJAAkACQAJAAkACQAJAAkAIQBpAHMATwBwAGUAbgAgACYAJgAgAGMAbABhAHMAcwBOAGEAbQBlACwACgAJAAkACQAJAAkACQAJACkAfQAKAAkACQAJAAkACQAJAD4ACgAJAAkACQAJAAkACQAJAHsAaQB0AGUAbQAuAHQAaQB0AGwAZQB9AAoACQAJAAkACQAJAAkAPAAvAHMAcABhAG4APgAKAAkACQAJAAkACQA8AC8ATABpAG4AawA+AAoACQAJAAkACQApACwACgAJAAkACQApAH0ACgAJAAkAPAAvAG4AYQB2AD4ACgAJACkAOwAKAH0ACgAKAGUAeABwAG8AcgB0ACAAZABlAGYAYQB1AGwAdAAgAFMAaQBkAGUATgBhAHYAOwAKAA%3D%3D)

### Expected result

it should not be error. It happen starting in 1.7.0, in 1.6.4 it's not happening.

### Code of Conduct

- [X] I agree to follow Biome's Code of Conduct

**Comment by Sec-ant** [comment]
The playground link is broken. Could you please provide the code and explain a bit about the error and expectation?

**Comment by ImBIOS** [comment]
@Sec-ant updated! 👍

**Comment by ematipico** [comment]
> @Sec-ant updated! 👍 

It's still broken

**Comment by ematipico** [comment]
This is a duplicate of https://github.com/biomejs/biome/issues/2468

**Comment by ematipico** [comment]
Duplicate of https://github.com/biomejs/biome/issues/2468
